### DISCUSSION TITLE
store only converted images

### DISCUSSION THREAD

**Discussion by pepeshark** [spatie/laravel-medialibrary#2361] [Q&A]
Hi,
can I store only the converted image? If yes, how?
To understand me:
I will have two image 10Mb original image and 3Mb converted. It is 13 Mb instead of 3Mb.

Controller:
```
$post->addMediaFromRequest('file')->toMediaCollection('post');
```
Model:
```    
public function registerMediaConversions($media = null): void
{
        $this->addMediaConversion('converted');
}
```

Do you have any suggestion?
Many thanks.

**Comment by pepeshark** [comment]
Do you have any suggestion?

**Reply by freekmurze** [reply]
You could write some code to delete the original one yourself, the package doesn't handle this case.
